src/focused-stack/index.ts

```diff
@@ -52,8 +52,9 @@ class FocusedKeyHandlerStack {
     }
     const key = this.getKeyCodeFromEvent(e);
 
-    const handlerGroup = this.stack[this.stack.length - 1];
+    const handlerGroup = this.stack[0];
```

Comment:
Why are we changing this?

As I understand this correctly we have to take the last added group handlers.
Am I wrong?

Reply:
Oh, this is because of useEffect fires from the bottom up! Got it!